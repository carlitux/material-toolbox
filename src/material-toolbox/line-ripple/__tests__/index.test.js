// @flow
import React from 'react';
import { shallow } from 'enzyme';
import LineRipple from '../index';

describe('component::LineRipple', () => {
  it('Should match snapshoot', () => {
    const component = shallow(<LineRipple />);
    expect(component).toMatchSnapshot();
  });
});
