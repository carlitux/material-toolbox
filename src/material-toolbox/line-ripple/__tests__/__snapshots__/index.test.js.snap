// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component::LineRipple Should match snapshoot 1`] = `
<div
  className="mdc-line-ripple"
  style={Object {}}
/>
`;
